con-base/src/main/java/ba/codecentric/base/domain/FacilityType.java

```diff
@@ -0,0 +1,44 @@
+package ba.codecentric.base.domain;
+
+import javax.persistence.*;
```

Comment:
What's an agreement about wildcard imports? I'm against them for the sake of readability and possible type name collisions.